lib/potassium/assets/app/frontend/utils/query-params.ts

```diff
@@ -0,0 +1,8 @@
+import { decamelizeKeys } from 'humps';
+import { stringify } from 'qs';
+
+export function serializeParams(params: Record<string, string | number | undefined>) {
```

Comment:
Maybe these types are too limited, it should be able to handle nested arrays and objects (for example when making requests to `Ransack`)

Reply:
True, changed it to `object`. Hope I didn't overcompensate